src/reducers/requests/posts.js

```diff
@@ -46,6 +46,17 @@ function getPostThread(state = initialRequestState(), action) {
     );
 }
 
+function getPostThreadWithRetryAttempt(state = 0, action) {
+    switch (action.type) {
+    case PostTypes.GET_POST_THREAD_WITH_RETRY_ATTEMPT:
+        return state + 1;
```

Comment:
Ok I'll add in a reset for the initial request. Those actions also have `cancel:true` in the offline meta which tells the middleware to remove any duplicate actions with the same type from the outbox.